utilities/four_c_utils/src/four_c_utils/check_test_files.py

```diff
@@ -88,16 +63,16 @@ def main():
 
     # error file (None for sys.stderr)
     errfile = args.out
-    errors = 0
-    allerrors = []
+
     # check input file tests
-    errors += check_inputtests(args.filenames, allerrors)
+    errors = check_inputtests(args.filenames)
 
-    utils.pretty_print_error_report("", allerrors, errfile)
-    return errors
+    utils.pretty_print_error_report("", errors, errfile)
+
+    if errors:
+        return 1
```

Comment:
```suggestion
        raise RuntimeError("Input files contain errors (see above)")
```
or, better, put the error message from `pretty_print_error_report` in the exception.